### PR TITLE
avatar menu

### PR DESCRIPTION
**Pull request by fuentesloic**

### CODE REVIEW

File: static/js/utils.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ function alert(type, message, delay) {
 
 window.onload = _ => {
   const menu = document.getElementById('menu')
-  const avatar = document.getElementById('avatarMenu')
+  const avatar = document.getElementById('avatar-img')
```
Comment (vinyll) [inline static/js/utils.js]: Are you sure? https://github.com/neomad-team/neomad.org/blob/master/templates/layout.html#L23

Comment (fuentesloic) [reply static/js/utils.js]: nop nop !!!!! I made a mistake

File: templates/layout.html [hunk 2]
```diff
@@ -52,4 +53,4 @@
 
 {% include 'user/localize.html' %}
 
-{% include '_analytics.html' %}
+{% include '_analytics.html' %}
```
Comment (vinyll) [inline templates/layout.html]: You should keep the new line at the end of the file.
https://www.google.com/search?q=end+of+file+newline

And your editor should always have it automatically: https://stackoverflow.com/questions/43052279/atom-github-removing-end-of-file-newline

**Review by vinyll — CHANGES_REQUESTED** [review]

File: templates/layout.html (hunk 2 above)
Comment (fuentesloic) [inline templates/layout.html]: you fix it in pro right ?

**Comment by vinyll** [comment]
ok, I re-added the newline in the end of the file.
Please @fuentesloic, make sure you set your editor to always have a newline in the end of your files (this is valid for all languages / all files).

**Review by vinyll — APPROVED** [review]

**Comment by fuentesloic** [comment]
Sorry I just check (some friends at home ...) my settings was `whitespace: false` so I change it (And I don't know why that change, a new package I think !). Anyway, I'm back with `whitespace`

**Pull request merged**